paasta_tools/tron_tools.py

```diff
@@ -415,15 +415,17 @@ def get_job_name(self):
     def get_action_name(self):
         return self.action
 
-    def get_secret_volumes(self) -> List[TronSecretVolume]:
+    def get_secret_volumes(self) -> List[TronSecretVolume]:  # type: ignore
```

Comment:
TypedDict does not support isinstance and issubclass for a subclass of a defined typeddict, mypy is sad

Reply:
rip, maybe we can unify TronSecretVolume and SecretVolume in the future (if we did it now we'd need another tron release, right?)